edx_exams/apps/api/serializers.py

```diff
@@ -97,6 +97,28 @@ class Meta:
         )
 
 
+class AssessmentControlReviewSerializer(serializers.ModelSerializer):
+    """
+    Serializer for AssessmentControlResult as a review. Currently a single
+    ACS response is equvalent to a review but this may change if we introduce
+    multiple ACS responses per attempt.
+    """
+
+    submission_time = DateTimeField(source='incident_time', format=None)
+    severity = serializers.DecimalField(max_digits=3, decimal_places=2)
+    submission_reason = serializers.CharField(source='reason_code')
+
+    class Meta:
+        """
+        Meta Class
+        """
+        model = AssessmentControlResult
+
+        fields = (
+            'submission_time', 'severity', 'submission_reason'
```

Comment:
I chose field names that make more sense to the UI as a 'review context' instead of just echoing the ACS model